cypress/integration/group2/organizations.ts

```diff
@@ -15,6 +15,8 @@
  */
 import { approvePotatoMembership, resetDB, setTokenUserViewPort } from '../../support/commands';
 
+const imageURL =
```

Comment:
I don't think the image data is being used (I could be wrong). 
But it is tempting for someone to rely on this someday and then for our tests to die if cloudinary is down
Just check this potato into Github (our tests would die anyway if GitHub was down)

Reply:
We don't actually have a test that checks whether the image is displayed or not. Meaning even if cloudinary is down, it would still pass.

Reply:
I'm aware, please review.